parm/parm_gdas/atm_crtm_coeff.yaml

```
@@ -0,0 +1,178 @@
+mkdir:
+- $(DATA)/crtm/
+copy:
+# Emissivity files
+- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISice.EmisCoeff.bin, $(DATA)/crtm/]
+- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff.bin, $(DATA)/crtm/]
```

```suggestion
- [$(HOMEgfs)/fix/gdas/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff.bin, $(DATA)/crtm/]
```
Here and everywhere in this yaml.

---

Change committed at c27618bb